Dockerfile

```diff
@@ -1,16 +1,21 @@
-FROM php:7.2-apache
+FROM node:10.11
 
-ENV HTML /var/www/html
-COPY source $HTML
+ENV HTML /opt/sinopia_profile_editor/
+WORKDIR $HTML
 
-RUN apt-get update && \
-    apt-get install -y gnupg curl && \
-    curl -sL https://deb.nodesource.com/setup_8.x | bash - && \
-    apt-get update && \
-    apt-get install -y nodejs
+COPY package*.json $HTML
+# FIXME: should this be npm install with --production flag?
+RUN cd $HTML && npm install
 
-RUN cd $HTML && \
-    npm install && \
-    npm install angular-local-storage && \
-    npm install -g grunt-cli && \
-    grunt
+# FIXME: do we want to run grunt locally and copy only dist/ to docker image?
```

Comment:
I feel like we want to leave in these steps in the Dockerfile just to insure that grunt has been run. What we could do if don't want these grunt artifacts in our final image is to remove then as the final steps in the RUN statement on line 13